.travis.yml

```diff
@@ -18,20 +18,20 @@ env:
     - PYTHONWARNINGS=all
     - SKIP_LATEX_BUILD=1
   matrix:
-    - DOCUTILS=0.12
```

Comment:
why dropping this?

Reply:
I just followed their own decision two owners (seems including you) made to keep the latest two versions to be used for tests as described below:

https://github.com/sphinx-doc/sphinx/pull/4050#issue-256475395

> - https://github.com/sphinx-doc/sphinx/pull/3217#discussion_r91833953
>  - the latest two versions are to be used for CI tests on Travis CI (except tox.ini)

Reply:
Oh, sorry. I'd completely forgotten it.
Okay, let's go forward :-)